notebooks/openvino/vision_language_quantization.ipynb

```diff
@@ -0,0 +1,842 @@
+   },
+   "source": [
+    "### Step 5b: Compare Model Sizes\n",
+    "\n",
+    "Now let's compare the file sizes of the original FP32 model and the quantized INT8 model:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "1eeaa81f-7fc5-49ba-80b8-2d95a1310a0c",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "def get_model_size(model_folder):\n",
+    "    model_size = 0\n",
+    "    for file in Path(model_folder).iterdir():\n",
+    "        if file.suffix==\".xml\":\n",
+    "            model_size += file.stat().st_size + file.with_suffix(\".bin\").stat().st_size\n",
+    "    model_size /= 1000 * 1000\n",
+    "    return model_size"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "3c862277",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "FP32 model size: 1028.25 MB\n",
+      "INT8 model size: 260.94 MB\n",
+      "INT8 size decrease: 3.94x\n"
+     ]
+    }
+   ],
+   "source": [
+    "fp32_model_size = get_model_size(fp32_model_path)\n",
+    "int8_model_size = get_model_size(int8_model_path)\n",
+    "print(f\"FP32 model size: {fp32_model_size:.2f} MB\")\n",
+    "print(f\"INT8 model size: {int8_model_size:.2f} MB\")\n",
+    "print(f\"INT8 size decrease: {fp32_model_size / int8_model_size:.2f}x\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7ef9297",
+   "metadata": {},
+   "source": [
+    "### 5c: Compare performance on different Intel Hardware platforms"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "94f18ce3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import time\n",
+    "from optimum.intel import OVModelForVisualCausalLM\n",
+    "\n",
+    "def benchmark(model, inputs, model_dir: str, nb_pass=10, warmup=4,max_tokens=50):\n",
+    "    \"\"\"\n",
+    "    Benchmark an OV visual causal LM model.\n",
+    "\n",
+    "    Returns a dict with:\n",
+    "    - avg_latency_sec\n",
+    "    - images_per_sec\n",
+    "    - tokens_per_sec\n",
+    "    - model_size_mb\n",
+    "    \"\"\"\n",
+    "    # --- Warmup ---\n",
+    "    for _ in range(warmup):\n",
+    "        _ = model.generate(**inputs)\n",
+    "\n",
+    "    # --- Timed inference ---\n",
+    "    start = time.time()\n",
+    "    for _ in range(nb_pass):\n",
+    "        outputs = model.generate(**inputs,max_new_tokens=max_tokens)\n",
+    "    end = time.time()\n",
+    "\n",
+    "    avg_latency = (end - start) / nb_pass\n",
+    "\n",
+    "    # --- Throughput calculations ---\n",
+    "    batch_size = inputs[\"pixel_values\"].shape[0] if \"pixel_values\" in inputs else 1\n",
+    "    num_tokens = outputs.shape[-1]  # sequence length\n",
+    "\n",
+    "    images_per_sec = batch_size / avg_latency\n",
+    "    tokens_per_sec = num_tokens / avg_latency\n",
```

Comment:
This is a bit of oversimplification, because such measurement does not take into account how long each actual model took to infer. For example if we increase the speed of vision encoder only, it will also influence the `token_per_sec` value. Perhaps it is ok for demo purposes though.

Reply:
@ezelanza please merge this one into your branch https://github.com/ezelanza/optimum-intel/pull/1